zerver/lib/event_schema.py

```diff
@@ -80,6 +80,10 @@
     ("stream_post_policy", int),
 ]
 
+stream_deletion_event_fields = [
```

Comment:
I guess this is not being used currently. We can either use this or just remove it.